src/software/estop/UARTCommunication.cpp

```diff
@@ -0,0 +1,67 @@
+
+#include "UARTCommunication.h"
+
+#include <boost/asio.hpp>
+#include <g3log/loglevels.hpp>
+#include <g3log/g3log.hpp>
+#include <iostream>
+#include <boost/function.hpp>
+#include <boost/exception/all.hpp>
+
+UARTCommunication::UARTCommunication(){}
+
+
+int UARTCommunication::openPort(io_service &ioService, int baudRate, std::string port)
+{
+    int uartCharacterSize = 8;
```

Comment:
Is this in bits? Also the code convention is that this is snake_case

Reply:
changed name to `uart_character_size_bits`